components/help.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useContext } from "react";
+
+import { LanguageContext } from "@contexts/translator";
+
+// eslint-disable-next-line react/prop-types
+const Help = ({ close }) => {
+  const { getTranslation } = useContext(LanguageContext);
+
+  return (
+    <div id="help-info" className="help-info padded spaced-lines">
+      <button
+        id="help-info-close-button"
+        className="help-info-close-button"
+        aria-label="Close help info section"
+        onClick={close}
+      >
+        X{" "}
+        <span
+        // data-translation-id="close"
+        >
+          {getTranslation("close")}
+        </span>
+      </button>
+      <p className="p txt-small">
+        <span
+          // data-translation-id="project_description"
+          // I don't love this but alas
```

Comment:
What does it do?

Reply:
We use some pretty nasty inline anchor tags in our translation here: https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/blob/f79a813b45c3cdd0b7d6fdd5c0b7b38fae2b6a80/src/i18n/eng.json#L30

React discourages writing unsanitized HTML so I had to use `dangerouslySetInnerHTML` in order to pass that in. We should probably just write Markdown in the translations instead and pass each translation through a parser, but that felt like too much work for this PR